gdxsv/lbs_discord.go

```diff
@@ -0,0 +1,524 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+var discordLiveStatusUpdateAvailable bool
```

Comment:
This will introduce race condition.
Use the atomic package or use chan for exclusive control.